lib/cylc/suite_srv_files_mgr.py

```diff
@@ -38,6 +39,16 @@ class SuiteServiceFileError(Exception):
     pass
 
 
+class SuiteCylcVersionError(Exception):
+    MESSAGE = "Suite Cylc version {} is incompatible with Cylc 7"
```

Comment:
```suggestion
    MESSAGE = "Suite Cylc version {} is not compatible with Cylc 7"
```

(Sounds grammatically cleaner to me, although I'm sure not everyone would agree)